### DISCUSSION TITLE
check web user credentials?

### DISCUSSION THREAD

**Discussion by gcrino** [telekom-security/tpotce#1529] [Q&A] [answered]
For some reason I cannot log into the T-pot landing page via browser. I have set up everything, my containers are running without problems and I already deployed my ssh via the deploy.sh script. I have tried to add new web users and log in with those credentials but my browser will not accept them. Is there a way to see what my user name and password is for the Web User? Or a way to perhaps redeploy my ssh? Any response or tips would be greatly appreciated!

Something I do want to add is that when I run the command "systemctl status ssh" it shows me that it is indeed active, however when I run the command "ssh -p '64297' '<user>@<my_ip>' it says: 
```
kex_exchange_identification: connection closed by remote host
Connection closed by <my_ip> port 64297
```

Any tips or help is appreciated! I am sorry if this seems like an obvious or silly question.

**Comment by t3chn0m4g3** [comment]
SSH runs on `tcp/64295` and the `deploy.sh` script does not setup SSH, but merely sets up the sensor to transmit logs via `https tcp/64294` to HIVE.
The only thing T-Pot changes in regard to SSH is to move the port to `tcp/64295`, all the users and such are the ones you set up with the system installation.

**Reply by gcrino** [reply]
ah okay, that makes more sense. Is there anything you recommend for troubleshooting regarding the T-pot landing page login? No matter how many times I try to login (with the correct credentials I set up) it still is failing to let me login. I can provide any additional information you may need.

**Reply by t3chn0m4g3** [reply]
This is a good [starting point](https://github.com/telekom-security/tpotce?tab=readme-ov-file#add-users-to-nginx-t-pot-webui).

**Reply by gcrino** [reply]
I have ran the genuser.sh script a few times, and none of the users I created worked. Looking into the .env file I do see that I have web user data there. Something I find odd is that when looking into the tpotce/data/nginx/conf/ directory, both nginxpasswd and lswebpasswd are blank. Are they supposed to be that way?

**Reply by t3chn0m4g3** [reply]
`nginxpasswd` is empty if you have configured the node to be a sensor, otherwise it is only empty if you have not restarted T-Pot after adding the user(s).
`lswebpasswd` only holds users if your T-Pot node is configured as `hive` with active `sensors`.

**Reply by gcrino** [reply]
Maybe a stupid question, but does configuring your node to be sensor or hive affect the web GUI at all? Also I appreciate the time you're taking out of your day to respond, I am very grateful

**Reply by t3chn0m4g3** [reply, marked as answer]
[Yes](https://github.com/telekom-security/tpotce?tab=readme-ov-file#distributed). The sensor does not offer any of the Landing Page tools.

**Reply by gcrino** [reply]
That did it! Switched to Hive and everything is working! Thank you so much for your time and support!

**Reply by t3chn0m4g3** [reply]
Great to hear. You are welcome 😄
